File: CPP/car.cpp
#include <iostream>
#include <vector>
#include <string>
#include "Vehicle.cpp"

using namespace std;

class Car : public Vehicle {
private:
    int numOfSeats;
    int numOfDoors;
public:
    Car(string plateNumber, string brand, int year, string color, int numOfSeats, int numOfDoors)
        : Vehicle(plateNumber, brand, year, color), numOfSeats(numOfSeats), numOfDoors(numOfDoors) {}

    void displayInfo() override {
        Vehicle::displayInfo();
        cout << "Number of Seats: " << numOfSeats << endl;
        cout << "Number of Doors: " << numOfDoors << endl;
    }
};

File: PHP/Car.php
<?php
require_once 'Vehicle.php';

class Car extends Vehicle {
    private $numOfSeats;
    private $numOfDoors;

    public function __construct($plateNumber, $brand, $year, $color, $numOfSeats, $numOfDoors) {
        parent::__construct($plateNumber, $brand, $year, $color);
        $this->numOfSeats = $numOfSeats;
        $this->numOfDoors = $numOfDoors;
    }

    public function displayInfo() {
        parent::displayInfo();
        echo "Number of Seats: " . $this->numOfSeats . PHP_EOL;
        echo "Number of Doors: " . $this->numOfDoors . PHP_EOL;
    }
}
?>
